brew/tailscale.rb

```diff
@@ -0,0 +1,87 @@
+# typed: false
```

Comment:
Should this include a license header?

Reply:
open question... brew did not require it or recommend it. I also spent time looking at specimens of brew formula in the wild to get a sense for what was the usual practice (for everything, not just licenses). having a license blurb comment directly inside the formula seems... rare -- definitely not the norm. I think for normal source code that constitutes the working functionality & I.P. of the Tailscale product a license blurb makes sense. but this file is intended to become a public recipe for endusers. a descendant of this file might end up in the main Homebrew collection.

I don't feel strongly about this either way, but my recommendation leaned to NO.